lang/cs/Org.Apache.REEF.Driver/Bridge/Events/ActiveContext.cs

```diff
@@ -44,48 +43,35 @@ public ActiveContext(IActiveContextClr2Java clr2Java)
         }
 
         [DataMember]
-        public string InstanceId { get; set; }
+        public string InstanceId { get; private set; }
+
+        private IActiveContextClr2Java Clr2Java { get; set; }
```

Comment:
I agree. But with these classes I never know whether they need to be this way for the bridge. @yingdachen Can you comment? They are also never set, a far as I can tell.